easybuild/easyconfigs/q/QUAST/QUAST-4.6.0-foss-2016b-Python-3.5.2.eb

```diff
@@ -0,0 +1,43 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'PythonPackage'
+
+name = 'QUAST'
+version = '4.6.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/ablab/%(namelower)s'
+description = """QUAST evaluates genome assemblies by computing various metrics.
+ It works both with and without reference genomes. The tool accepts multiple
+ assemblies, thus is suitable for comparison."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/ablab/%(namelower)s/archive/']
+sources = ['%(namelower)s_%(version)s.tar.gz']
+
+checksums = ['6bee86654b457a981718a19acacffca6a3e74f30997ad06162a70fd2a181ca2e']
+
+install_target = 'install_full'
+
+dependencies = [
+    ('Python', '3.5.2'),
+    ('Perl', '5.24.0'),
+    ('matplotlib', '1.5.3', versionsuffix),
+    ('Java', '1.8.0_131', '', True),
+    ('Boost', '1.63.0', versionsuffix),
+]
+
+# bypass import check, sanity check fails without this
+# 'ImportError: No module named quast' during sanity check
+options = {'modulename': 'site'}
```

Comment:
Are there no Python installed? There seem to be, since you're checking for a non-empty `lib/python%(pyshortver)s/site-packages/`?

If there are, please use a 'proper' module name here rather than `site` which is always there.

Reply:
The `site-packages` directory only contains this:

```
$ ls /easybuild/test/software/QUAST/4.6.0-foss-2016b-Python-3.5.2/lib/python3.5/site-packag
es/
easy-install.pth  joblib-0.11-py3.5.egg  __pycache__  quast-4.6.0-py3.5.egg  simplejson-3.13.2-py3.5-linux-x86_64.egg  site.py
```

So I don't think that we can provide a real module to check for.

Reply:
I think you need to check for `import quast_libs`, so try this (and drop the comment above):

```python
options = {'modulename': 'quast_libs'}

Reply:
Thanks, that worked.